docs/BUILD

```diff
@@ -43,6 +43,36 @@ filegroup(
     srcs = glob(["root/**/*.pb"]),
 )
 
+genrule(
+    name = "api_rst",
+    srcs = ["@envoy_api//:v3_protos"],
+    outs = ["api_rst.tar"],
+    cmd = """
+    OUTDIR=$$(mktemp -d) \
+    && XDS_PROTO_SRCS=$$(cat $(location :xds_proto_srcs) | cut -d/ -f3- | tr ':' '/') \
```

Comment:
With this level of complexity, prefer moving to dedicated `.sh` with comments etc.